build.sh

```diff
@@ -25,7 +25,7 @@ function usage()
 # Determine how many parallel jobs to use for make based on the number of cores
 unamestr="$(uname)"
 if [[ "$unamestr" == "Linux" ]]; then
-  PARALLEL=$(nproc)
```

Comment:
@ericl 
Do you remember the reason for changing `nproc` to `nproc  --all`?

Reply:
nproc --all doesn't respect the CPU count within containers I believe, which was overloading jenkins

Reply:
`nproc --all` shows the CPU count of the real physical machine for containers rather than the virtual CPU count of the containers, right?

In our case, there is a contianer with 8 virtual CPUs, and the num of the CPUs of its host machine is 32. The result of `nproc --all` is 32, so `make -j 32` will run out of memory because of too many parallel jobs.

It's better to get the virtual CPU count here I think.

Reply:
Ah, I remember now. The reason is that on some AWS machines (the deep learning AMI, at least), nproc returns 1 no matter how many cores are available.

I think this is fine to change back to nproc, since the bazel build will presumably fix this anyways.

Reply:
Thanks @ericl .
I'll change it back soon.